test/ptp/ptp.go

```diff
@@ -85,72 +86,28 @@ var _ = Describe("[ptp]", func() {
 
 	Describe("PTP e2e tests", func() {
 		var ptpRunningPods []v1core.Pod
+		var masterNodeLabel, slaveNodeLabel string
 
 		execute.BeforeAll(func() {
-			ptpconfigList, err := client.Client.PtpConfigs(PtpLinuxDaemonNamespace).List(metav1.ListOptions{})
-			Expect(err).ToNot(HaveOccurred())
 
-			for _, ptpConfig := range ptpconfigList.Items {
-				if ptpConfig.Name == PtpGrandMasterPolicyName || ptpConfig.Name == PtpSlavePolicyName {
-					err = client.Client.PtpConfigs(PtpLinuxDaemonNamespace).Delete(ptpConfig.Name, &metav1.DeleteOptions{})
-					Expect(err).ToNot(HaveOccurred())
-				}
+			if !discovery.Enabled() {
+				configurePTP()
 			}
+			masterConfigs, slaveConfigs := discoveryPTPConfiguration(PtpLinuxDaemonNamespace)
 
-			nodeList, err := client.Client.Nodes().List(metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=", PtpGrandmasterNodeLabel)})
-			Expect(err).ToNot(HaveOccurred())
-			for _, node := range nodeList.Items {
-				delete(node.Labels, PtpGrandmasterNodeLabel)
-				_, err = client.Client.Nodes().Update(&node)
-				Expect(err).ToNot(HaveOccurred())
+			if len(masterConfigs) == 0 {
```

Comment:
Following yesterday's discussion, I think we need this only for https://github.com/openshift/ptp-operator/blob/a8f94f9591dba78c4bd7fe7a2cc8feb5cb55927e/test/ptp/ptp.go#L228
Only that test should skip if master is not found